src/components/containers/ProfileSidebar/ProfileSidebar.jsx

```diff
@@ -0,0 +1,74 @@
+/* eslint-disable no-console */
+/* eslint-disable react/prop-types */
+import React, { Component } from 'react';
+import { Icon } from 'semantic-ui-react';
+
+import Avatar from '../../presentationals/Avatar/Avatar.jsx';
+import Button from '../../presentationals/Button/Button.jsx';
+
+import './ProfileSidebar.scss';
+
+class ProfileSidebar extends Component {
+  constructor(props) {
+    super(props);
+  }
+  render() {
+    let userAvatar = {
+      url:
+        this.props.profile.image ||
+        'https://res.cloudinary.com/nedy123/image/upload/v1557149386/avatar_pupqrh.svg',
```

Comment:
let put this image url in our config file.

Reply:
I have moved the URL into the config.js file